### DISCUSSION TITLE
Traceless open a hidden volume

### DISCUSSION THREAD

**Discussion by solokot** [hardcore-sushi/DroidFS#159] [Ideas]
Hello!
After changing the interface in terms of opening volumes, the application is now forced to remember paths to encrypted volumes, which is convenient, but hidden volumes have almost lost their purpose (the possibility of a credible denial of the fact of their presence at least on a routine inspection of the device). Constantly deleting an entry from the volume list is inconvenient at the very least.
I suggest we add a "+" button action by long pressing "open hidden volume", i.e. open the volume name and password prompt window, and then close the volume without creating records for it.

**Comment by hardcore-sushi** [comment]
Yes you're right, what about adding a switch under the "Volume name" entry named "Remember volume" which when turned off will not register the volume in the home page list?

**Reply by solokot** [reply]
Yes, remembering the registration status of the volume on the home screen is also the right solution.

**Comment by hardcore-sushi** [comment]
Since 7c2f87109a8db7579587958ba0e3e3bdc21eabc2 you can create and open encrypted volumes without the app remembering. This feature will be available in the next DroidFS release.
